funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -437,10 +446,17 @@ def stop(self):
         self._kill_event.set()
         self._quiesce_event.set()
 
+    def handle_sigterm(self, sig_num, curr_stack_frame):
+        logger.info("Saving unacked results to disk")
```

Comment:
Clarify that the endpoint received a sigterm message. It would also be useful to also have a log message that the persist succeeded.